### PR TITLE
Update about-code-owners.md

### PR DESCRIPTION
**Pull request by rajbos**
### Why:
Make the DO NOT part more clear: it's super easy to read over it and start to implement the patterns that do not work.
We can do it this way or add it into a NOTE block

### What's being changed (if available, include any code snippets, screenshots, or gifs):
Text on the docs page.

### Check off the following:

- [ ] I have reviewed my changes in staging (look for the "Automatically generated comment" and click the links in the "Preview" column to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Comment by rajbos** [comment]
It did not stand out to much with making it bold (on the dark theme) so I made it italic.

## Bold:
image

## Italic
Waiting for the deployment update

**Comment by cmwilson21** [comment]
👋 @rajbos Thanks so much for opening a PR! I'll get this triaged for review :zap:

**Comment by cmwilson21** [comment]
👋 @rajbos, I spoke with the team and we agree that this part should stand out a bit more. However, instead of using italics, a warning box may be more appropriate. Would you be up for removing the header on line 119 and enclosing this section in a [warning box](https://github.com/github/docs/blob/main/contributing/content-style-guide.md#formatting-callouts)?

**Review by rajbos — COMMENTED** [review]
Handling review comment

**Comment by rajbos** [comment]
Done. Hopefully this is the correct syntax for it

**Comment by cmwilson21** [comment]
@rajbos I don't see any changes in the commits. Just checking that the push to this branch was successful? 🤔

**Comment by tarek295197** [comment]
😊🤭 I'm sorry 
Thank you very much 😊

**Comment by rajbos** [comment]
Hmm, not sure what happened there (I used the web interface!). They should be in this time 🙌 .

File: content/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners.md [hunk 1]
```diff
@@ -116,11 +116,14 @@ apps/ @octocat
...
-### Syntax exceptions
-There are some syntax rules for gitignore files that do not work in CODEOWNERS files:
+
+{% warning %}
+**Warning:** Syntax exceptions
+There are some syntax rules for gitignore files that *do not work* in CODEOWNERS files:
 - Escaping a pattern starting with `#` using `\` so it is treated as a pattern and not a comment
 - Using `!` to negate a pattern
 - Using `[ ]` to define a character range
+{% endwarning %}
```
Comment (cmwilson21) [inline content/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners.md:126]:
```suggestion
**Warning:** There are some syntax rules for gitignore files that *do not work* in CODEOWNERS files:
- Escaping a pattern starting with `#` using `\` so it is treated as a pattern and not a comment
- Using `!` to negate a pattern
- Using `[ ]` to define a character range

{% endwarning %}
```

**Review by cmwilson21 — APPROVED** [review]
Looks great!

**Comment by cmwilson21** [comment]
@rajbos Thanks for bringing this up and making the changes! I made just a couple slight tweaks to get the markdown to render properly, but it looks good! 

I'll get this merged once all the tests have passed 🍏 

Thanks for another great contribution to the docs! 💖

**Pull request merged**